jobs/JGDAS_GLOBAL_MARINE_ANALYSIS_LETKF

```diff
@@ -12,9 +12,14 @@ GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
 gPDY=${GDATE:0:8}
 gcyc=${GDATE:8:2}
 
+export GDUMP=${GDUMP:-"gdas"}
+export GDUMP_ENS="enkf${GDUMP}"
```

Comment:
where are `GDUMP` and `GDUMP_ENS` used?  I see it defined here, but not used.

Reply:
These get used in the jinja yamls used for specifying the background files for staging

Reply:
I don't see it in the atm or snow applications needing it. 
Surely they also need a variable such as this.

Reply:
The `GDUMP`/`GDUMP_ENS` is obsolete here, only one is needed. In atm it seems to get hardcoded in a local config; is that the way?

Reply:
Yes. It is very likely GDUMP for analysis is based on the late cycle.

Reply:
removed